static/app/utils/performance/segmentExplorer/tagTransactionsQuery.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import {MetaType} from 'app/utils/discover/eventView';
+import GenericDiscoverQuery, {
+  DiscoverQueryProps,
+} from 'app/utils/discover/genericDiscoverQuery';
+import withApi from 'app/utils/withApi';
+
+export type TableDataRow = {
+  id: string;
+  [key: string]: React.ReactText;
```

Comment:
To the TableDataRow? Yeah the other queries will have almost the same format, I think it's better to not mix implementations though since these are meant to be isolated from each other.